monkestation/code/modules/client/verbs/multi_character.dm

```diff
@@ -0,0 +1,191 @@
+			return
+		usr.client.cmd_assign_mob_to_client(src)
+
+/client/proc/cmd_assign_mob_to_client(mob/mob_to_assign in GLOB.mob_list)
+	set category = "Admin.Game"
+	set name = "Assign Mob to Client"
+
+	// Just in case someone convinces the server to somehow call this...
+	if(!check_rights(NONE))
+		return
+
+	if(!mob_to_assign)
+		// How did you even get here...?
+		CRASH("Assign Mob to Client verb called without a client.")
+
+	// Ask which client to assign this mob to
+	var/client/assign_to = input(src, "Assign this mob to which admin client?", "Assigning mob...") as null|anything in sort_list(GLOB.admins)
+	if(!assign_to)
+		return
+
+	// If the client we're assigning this mob to isn't an admin (somehow), alert the admin of this
+	// and refuse to continue.
+	if(!assign_to.holder)
+		tgui_alert(
+			src,
+			title = "Not an Admin",
+			message = "The client you selected is (somehow) not an admin."
+		)
+		return
+
+	// If the client is already assigned this mob, alert the admin of this and refuse to continue.
+	if(LAZYFIND(assign_to.mobs_assigned, mob_to_assign))
+		tgui_alert(
+			src,
+			title = "Mob Already Assigned",
+			message = "This mob is already assigned to the client you selected.",
+		)
+		return
+
+	// If the mob is already being controlled by a different client than we selected, ensure the
+	// admin is aware that they will not be immediately able to take control.
+	if(mob_to_assign.ckey && !mob_to_assign.is_controlled_by_client(assign_to))
+		var/understands = tgui_alert(
+			src,
+			title = "This Mob Is Already Under Control",
+			message = "This mob is already being controlled by [mob_to_assign.ckey]. If you assign this mob to [assign_to], they will not be able to switch to this mob until the previous controller relinquishes control. Do you wish to continue?",
+			buttons = list("Yes", "No"),
+		)
+		if(understands != "Yes")
+			return
+
+	// Send an admin message...
+	message_admins(span_adminnotice("[key_name_admin(src)] assigned a mob ([ADMIN_LOOKUPFLW(mob_to_assign)]) to [key_name_admin(src)]."))
+	// Notify the acting admin of how to undo this action...
+	to_chat(src, span_notice("You have granted [assign_to] control of a mob. To undo this action, var-edit the 'mobs_assigned' list contained here: [ADMIN_VV(assign_to)]."))
+
+	// Add the mob to the client's assigned mob list...
+	LAZYOR(assign_to.mobs_assigned, mob_to_assign)
+	// and to the mob's history of clients it's been assigned to
+	LAZYOR(mob_to_assign.client_assignment_history, assign_to.ckey)
+
+	// ...and notify the user that they've been granted control.
+	to_chat(assign_to, span_notice("You have been granted control of a mob. Use the Switch Character verb in the OOC tab to switch to the mob."))
+
+/// Verb given to clients which allows them to instantly swap between any characters assigned to
+/// them.
+// TODO: Probably add an `assigned_to` var to mobs, and use that to check if the mob is already assigned
+/client/verb/switch_character()
```

Comment:
Since mobs can only be assigned to admin clients anyways, this should just be a `/client/proc/switch_character()` thats only given to admins, to avoid possible confusion.

Reply:
I have no plans to continue this PR since I don't currently feel like working on Monke's repo - so if you want to copy the PR and make the changes yourself, feel free.

Reply:
Ah, sorry for the bother, then.

Reply:
Ah don't worry, I just wanted to let you know that I don't plan to work on this functionality.